Asterisk.2013/Asterisk.NET/Manager/Event/JoinEvent.cs

```diff
@@ -15,7 +15,7 @@ public JoinEvent(ManagerConnection source)
         ///     Get/Set the Caller*ID number of the channel that joined the queue if set.
         ///     If the channel has no caller id set "unknown" is returned.
         /// </summary>
-        public string CallerId { get; set; }
+        public string CallerIdNum { get; set; }

```

Comment:
This would not be backward compatible.
I agree that we need to get this done at some point. So doing it for v2 is likely ok.